### PR TITLE
MOB-4141 post release 1.6.0

### PR DESCRIPTION
**Pull request by mateid-kr**
## JIRA
[MOB-4141](https://karhoo.atlassian.net/browse/MOB-4141)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [X] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [X] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/fragment/CheckoutFragment.kt [hunk 1]
```diff
@@ -172,6 +173,28 @@ internal class CheckoutFragment : Fragment() {
...
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        savedInstanceState?.let {
+            if(it[PASSENGER_DETAILS] != null){
+                val passDetails = it[PASSENGER_DETAILS] as PassengerDetails
+                checkoutView.bindPassenger(passDetails)
+            }
+//            if(it[SAVED_PAYMENT_INFO] != null){
```
Comment (dianakarh) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/fragment/CheckoutFragment.kt:190]: Is there any reason to keep this commented code?

File: uisdk/src/main/res/values-en-rGB/strings.xml [hunk 2]
```diff
@@ -125,18 +124,10 @@
     <!-- FEEDBACK -->
     <string name="kh_uisdk_feedback_title">Please, tell us more about your trip</string>
     <string name="kh_uisdk_fees_and_taxes_included">Taxes and fees included</string>
-    <string name="kh_uisdk_filter">Filter</string>
```
Comment (dianakarh) [inline uisdk/src/main/res/values-en-rGB/strings.xml:128]: where these removed for the new quote list by any chance and were not moved to the feature branch?

File: uisdk/src/main/res/values-en-rGB/strings.xml [hunk 3]
```diff
@@ -270,8 +261,11 @@
     <string name="kh_uisdk_network_error">We seem to be having issues connecting to the network</string>
     <string name="kh_uisdk_new_ride">NEW RIDE</string>
     <string name="kh_uisdk_next">Next</string>
+    <string name="kh_uisdk_no_availability">We’re not live here yet! Get in touch if you have suggestions for local fleets.</string>
+    <string name="kh_uisdk_no_availability_subtitle">Please select another time</string>
+    <string name="kh_uisdk_no_availability_title">No availability for the requested time</string>
     <string name="kh_uisdk_no_destination">No Destination</string>
-    <string name="kh_uisdk_no_results_found">Apologies ASAP bookings are not currently available for this destination, however you can book in advance for your next trip!</string>
```
Comment (dianakarh) [inline uisdk/src/main/res/values-en-rGB/strings.xml:274]: These values should be the other way around. The long text should be the one to stay. Valid for all languages

Comment (mateid-kr) [reply uisdk/src/main/res/values-en-rGB/strings.xml:274]: The long text was moved above in the K3002 string

File: uisdk/README.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ implementation 'com.github.karhoo:karhoo-android-ui-sdk:develop-SNAPSHOT'
 ## Initialisation
 
 There are a few things the UI SDK needs to know before you can get started such as what environment to connect to, or what kind of authentication method to use.
-To configure the SDK you will need to provide an implementation of our KarhooUISDKConfiguration interface. This lets our SDK grab certain dependencies and configuration settings.
+To configure the UISDK from version 1.5.7 and above, you will need to provide an implementation of our KarhooUISDKConfiguration interface (e.g. KarhooConfig: KarhooUISDKConfiguration) and the desired payment provider
```
[the remark this answers is not in the document]
Comment (mateid-kr) [reply uisdk/README.md:30]: Good catch

File: uisdk/src/main/res/values-en-rGB/strings.xml (hunk 2 above)
Comment (mateid-kr) [reply uisdk/src/main/res/values-en-rGB/strings.xml:128]: On develop we were having the translations which were related to the new quote list branch. I've removed them both from develop and master

**Pull request merged**